### PR TITLE
Update to userinfo section

### PR DESCRIPTION
**Pull request by rebeksburnett**
Most of this is superficial, but I had trouble following the instructions for getting the userinfo, so I added a bit more to that section.

In the yaml file, the description for the userinfo access token seemed to refer to the "regular access token", but it needs a token from the Login API instead. So I reused the description from the Login API access token section. 

I found out that you should NOT include the subscription key in the header for the userinfo and login calls, and I'd added those in to the postman, so I'm removing those here.

### CODE REVIEW

File: tools/vipps-ecom-api-postman-collection.json [hunk 1]
```diff
@@ -45,11 +45,6 @@
 								"value": "application/x-www-form-urlencoded",
 								"type": "text"
 							},
-							{
```
Comment (rebeksburnett) [inline tools/vipps-ecom-api-postman-collection.json:48]: I added these here, but that turned out to be wrong. I now know how to run the postman collections, so I will be able to test them next time I fiddle with them.

File: docs/swagger.yaml [hunk 2]
```diff
@@ -1098,10 +1098,8 @@ paths:
         - in: header
           name: Authorization
           description: >-
-            The access token is a base64-encoded string that is required for all API calls.
```
Comment (rebeksburnett) [inline docs/swagger.yaml:1101]: This comes from the login api access token description. The token used here is not the one that we get from the accesstoken/get endpoint

File: vipps-ecom-api.md [hunk 3]
```diff
@@ -1566,28 +1567,30 @@ Example of the userInfo flow:
...
-4. Retrieve the `sub` by calling
-   [`GET:/ecomm/v2/payments/{orderId}/details`](https://vippsas.github.io/vipps-ecom-api/#/Vipps%20eCom%20API/getPaymentDetailsUsingGET)
-5. Using the sub from step 4, call
+   [`POST:/accesstoken/get`](https://vippsas.github.io/vipps-ecom-api/#/Authorization%20Service/fetchAuthorizationTokenUsingPost).
+2. Add `scope` to the
+   [`POST:/ecomm/v2/payments`](https://vippsas.github.io/vipps-ecom-api/#/Vipps%20eCom%20API/initiatePaymentV3UsingPOST) call and include the scopes you need access to (e.g., "name address email phoneNumber birthDate"). Separate scopes with spaces.
+3. The user consents to sharing the information and performs the payment in Vipps.
+4. You retrieve the user identifier, `sub`, by calling
+   [`GET:/ecomm/v2/payments/{orderId}/details`](https://vippsas.github.io/vipps-ecom-api/#/Vipps%20eCom%20API/getPaymentDetailsUsingGET).
+5. Retrieve a login token via the Vipps Login API:
```
Comment (rebeksburnett) [inline vipps-ecom-api.md:1588]: Added a new step here. You have to get this login token for the next call to work.

File: docs/swagger.yaml (hunk 2 above)
Comment (tht13) [reply docs/swagger.yaml:1101]: are you sure? I thought we changed this to use the normal ecom auth

Comment (tht13) [reply docs/swagger.yaml:1101]: That used to be the case but this changed about 12 months ago
It now used normal ecom token

**Comment by rebeksburnett** [comment]
I was trying to update this PR with "git pull --rebase origin master". Did I do that right? I didn't expect to see all the commits that have come in from master listed out.

File: docs/swagger.yaml (hunk 2 above)
Comment (rebeksburnett) [reply docs/swagger.yaml:1101]: @tht13 - Can we look at the postman together this week and get it set up correctly?

**Pull request closed without merging**